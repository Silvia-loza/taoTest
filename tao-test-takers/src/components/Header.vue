<template>
  <header class="header">
   <img id=logoHeader alt="Tao Testing logo" src="../assets/TAOTesting.png">
    <ul>
      <li>Product</li>
      <li>Solutions</li>
      <li>Services</li>
      <li>Support</li>
      <li>About Us</li>
      <li>Resources</li>
      <li>Blog</li>
    </ul>
  </header>
</template>

<script>
export default {
  name: 'Header',
  props: {
  }
}
</script>

<!-- Add "scoped" attribute to limit CSS to this component only -->
<style scoped>
*{
color: #bc262d;
margin: 0 auto;
box-sizing: border-box;
}

#logoHeader{
justify-content: flex-start;
width:10%;
margin-top: 2%;
margin-bottom: 2%;
}

.header {
  height: inherit;
  width: inherit;
  display: flex;
  justify-content: space-between;
  align-items: center;
  box-shadow: 0.5px 9px 9px 4px rgba(0, 0, 0, 0.2);
   }
   ul {
   width:60%;
  }
   li {
    margin: 0 3%;
    font-size: 120%;
   }
</style>
