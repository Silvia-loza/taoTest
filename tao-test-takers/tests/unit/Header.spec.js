import { shallowMount } from '@vue/test-utils'
import Header from '@/components/Header.vue'

describe("Header works", () => {
    it("renders the component", () => {

        //arrange item to be mounted
        const wrapper = shallowMount(Header)

        //assert that the component is mounted and that it matches the snapshot creted, will inform us about changes.
        expect(wrapper.html()).toMatchSnapshot()
    })
})