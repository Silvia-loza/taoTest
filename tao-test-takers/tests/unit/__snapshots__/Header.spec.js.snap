// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header works renders the component 1`] = `
<header class="header"><img id="logoHeader" alt="Tao Testing logo" src="">
  <ul>
    <li>Product</li>
    <li>Solutions</li>
    <li>Services</li>
    <li>Support</li>
    <li>About Us</li>
    <li>Resources</li>
    <li>Blog</li>
  </ul>
</header>
`;
